core/print_api.php

```diff
@@ -2006,6 +2008,23 @@ function print_bug_attachment_preview_image( array $p_attachment ) {
 	echo '</a></div>';
 }
 
+/**
+ * Prints the preview of an audio file attachment.
+ * @param array $p_attachment An attachment array from within the array returned by the file_get_visible_attachments() function.
+ * @return void
+ */
+function print_bug_attachment_preview_audio( array $p_attachment ) {
+	$t_file_url = $p_attachment['download_url'] . '&show_inline=1' . form_security_param( 'file_show_inline' );
```

Comment:
It would be great to update the `print_bug_attachment_header` function to show font-awesome icon `fa-file-audio-o` for audio files.
https://fontawesome.com/v4.7.0/icon/file-audio-o

Reply:
done.